.github/workflows/tests.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - run: pip install virtualenv
       - run: pip install wheel
       - name: pip install
-        run: pip install -r requirements.txt
+        run: pip install -r requirements.txt -r requirements-docs.txt
```

Comment:
@stevepiercy just guessing. This could be the offending change that made the CI build fail. I still have no clue how that triggered the missing mockup. Though, we had numerous occasions in Plone 6 already where packages were missing test dependencies.

Reply:
I think your PR proves that these changes had no effect.
https://github.com/plone/plone.restapi/pull/1350